wordclock/http_server.c

```diff
@@ -746,6 +746,10 @@ static void handle_clock_cfg_post(int s, wificfg_method method,
             }
         }
     }
+    if (g_settings.textEffect == TEXTEFFECT_RANDOM) {
```

Comment:
Dit hoeft niet in de http server toch?
Je ziet het als een filter dus dan zorgt AlsApplyTextEffect er voor dat de juiste kleur wordt ingesteld

Reply:
Dit is ervoor om direct bij het opslaan van de opties een random kleur te kiezen. Het filter zorgt ervoor dat het volgende uur een nieuwe random kleur wordt gekozen.